src/geometries/ParametricGeometry.js

```diff
@@ -63,6 +63,12 @@ function ParametricBufferGeometry( func, slices, stacks ) {
 
 	var i, j;
 
+	if ( func.length < 3 ) {
+
+		console.error( 'Parametric geometries now require modification of a third THREE.Vector3 argument.' );
```

Comment:
How's this?

```js
console.error( 'THREE.ParametricGeometry: Function must now support a THREE.Vector3 as third parameter.' );
```

Reply:
The error message applies to both ParametricGeometry and ParametricBufferGeometry right now, unless you want it to imply the former only. And I think using the word "modify" rather than "support" more accurately describes why the failure occurs.